### PR TITLE
Allow geomap preprocessor to work without Table import. 

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description
Also fix invalid viewport position in some cases.
Fix version of deckl used to avoid future issues with updates.

### Checklist
Please include the following checklist in your PR:

- [ ] ~The `CHANGELOG.md` was updated with the changes introduced in this PR.~
- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/enso-org/enso/blob/main/docs/style-guide/rust.md) style guide.
- [x] All code has automatic tests where possible.
- [x] All code has been profiled where possible.
- [x] All code has been manually tested in the IDE.
- [ ] ~All code has been manually tested in the "debug/interface" scene.~
- [x] All code has been manually tested by the PR owner against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).
- [x] All code has been manually tested by at least one reviewer against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).

### CODE REVIEW

File: src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js [hunk 1]
```diff
@@ -142,13 +142,23 @@ class GeoMapVisualization extends Visualization {
 
     onDataReceived(data) {
         if (!this.isInit) {
+            // FIXME: This should be simplified when issue [#1167]
+            //  (https://github.com/enso-org/ide/issues/1167) has been implemented.
+            //  Use the previous version from the history again.
```
Comment (wdanilo) [inline src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js]: Please add comment to that issue mentioning the problem

Comment (wdanilo) [inline src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js]: Also, Id keep old version in comment, as we plan to revert it soon (the mentioned issues is high-priority)

**Review by wdanilo — APPROVED** [review]

File: src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js (hunk 1 above)
Comment (MichaelMauderer) [reply src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js]: Done.

Comment (MichaelMauderer) [reply src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js]: Done.

**Review by farmaazon — CHANGES_REQUESTED** [review]
I've spotted some bug during tests:
1. Create `Geo.point` passing huge latitude or longitude
2. Show GeoMap visualization. Nothing is shown, the console prints error that the latitude is too large. So far it's ok.
3. Change the huge value to some normal

At this point the visualization is still not displayed - the console shows errors about "Cannot read property 'getLayers' of null"

When I switch to other visualization and switch back, I see the vis working properly. However the error log about 'getLayers' is still appearing on mouse events - likely we have some leak here.

**Review by farmaazon — APPROVED** [review]

**Pull request merged**